### ISSUE TITLE
Nether Worm Fossil

### ISSUE THREAD

**Issue by TheDeathlyCow** [TheDeathlyCow/scorchful#21]
Large fossils should generate in The Nether of giantic Nether Worms. These should be circular bone shapes that appear in all Nether biomes, representing the jaws/teeth of the ancient worms. Obvious reference to Dune.
